### DISCUSSION TITLE
Allow merging of multiple BoMs

### DISCUSSION THREAD

**Discussion by dashmesh** [DependencyTrack/dependency-track#899] [Q&A] [answered]
Hi Steve, 
Is it possible to have Dependency Track handle multiple BoM files per application if one is unable to have an aggregate BoM? 
Is there a tool that you are aware of that helps with BoM file aggregation?
Thanks,
Dashmesh

**Comment by stevespringett** [comment, marked as answer]
DT doesn't support merging multiple BOMs, but there is an enhancement request open to support uploading multiple BOMs at the same time and aggregating the inventory.
https://github.com/DependencyTrack/dependency-track/issues/130

The NPM implementation does have the ability to merge - although its limited and has known issues, it does work. The CycloneDX CLI will eventually have this behavior as well and will be the preferred way of achieving this once complete.

https://github.com/CycloneDX/cyclonedx-cli
